src/sentry/tasks/derive_code_mappings.py

```diff
@@ -104,7 +103,7 @@ def derive_code_mappings(
         return
 
     installation, organization_integration = get_installation(org)
-    if not installation:
+    if not installation or not organization_integration:
```

Comment:
@armenzg could you review the changes here to make sure this is a safe assumption to make?

Reply:
This is great! Thank you!